### PR TITLE
Refs [TASK][25431] Initialize Project 

### PR DESCRIPTION
**Pull request by hoatb-1022**
https://edu-redmine.sun-asterisk.vn/issues/25431

### CODE REVIEW

**Comment by hoatb-1022** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require bootstrap
+ *= require_tree .
+ *= require_self
+ */
+
+@import "bootstrap";
```
Comment (manhnt-0960) [inline app/assets/stylesheets/application.scss:18]: trên có require bootstrap rồi mà e

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>InternRubyKindmateClone</title>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb:4]: i18n

**Comment by hoatb-1022** [comment]
Screen Shot 2020-07-08 at 08 35 27

**Comment by hoatb-1022** [comment]
ready

**Pull request merged**